Clinical Key.js

```diff
@@ -130,6 +140,117 @@ function doWeb(doc, url){
 		search.setSearch({ ISBN: isbn });
 		search.getTranslators();
 	}
+	else if (contentType == 'journalArticle') {
+		eid = url.split('/');
+		pii = eid.pop().slice(7);
+		
+		/* can use xpath to get pii; guessing the url is faster
+		var eid = ZU.xpathText(doc, "//ul/@data-eid");
+		pii = eid.slice(7);*/
+		
+		if(!/^S(\d{15}X|\d{16})/.test(pii)){
+			throw new Error('PII incorrectly formatted');
+			return;
+		}
+		
+		doSearch(pii, doc);
+	}
+	
+}
+
+//Search & Processing based on CrossRef.js translator
+function doSearch(pii, doc){
+	crossRefQuery = 'http://api.crossref.org/works?query=' + pii
+	Zotero.Utilities.HTTP.doGet(crossRefQuery, function(responseText) {
```

Comment:
This sounds similar to the lines from ORCID translator, see https://github.com/zotero/translators/blob/master/ORCID.js#L68-L76 which we currently use. (The `minVersion` has to be then `4.0.29.11`.) However, I haven't look into the details here...